### PR TITLE
fix(i18n): Fixed syntax

### PR DESCRIPTION
**Pull request by rakekniven**

### CODE REVIEW

**Comment by Pytal** [comment]
This syntax is invaid because the prop value is already surrounded by double quotes, we need to move these translations into data properties instead @rakekniven

**Comment by Pytal** [comment]
Or could just keep it where it is and escape like `\'` in the string @rakekniven

File: src/components/ShareAccess.vue [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
+		<h3>{{ t('privacy', 'People you\'ve shared with') }}</h3>
 
 		<NcLoadingIcon v-if="isLoading"
-			:name="t('privacy', "Loading people you've shared with …")"
+			:name="t('privacy', 'Loading people you\'ve shared with …')"
 			:size="40" />
 
 		<p v-else-if="!uniqueShareUIDs.length">
-			{{ t('privacy', "You don't have any shares with individual users.") }}
+			{{ t('privacy', 'You don\'t have any shares with individual users.') }}
```
Comment (Valdnet) [inline src/components/ShareAccess.vue:31]:
```suggestion
			{{ t('privacy', 'You do not have any shares with individual users.') }}
```

**Review by Valdnet — COMMENTED** [review]
Standardization at Transifex 😉.

**Review by Pytal — APPROVED** [review]

**Pull request merged**